pkg/kubelet/nodestatus/setters.go

```diff
@@ -295,16 +294,15 @@ func MachineInfo(nodeName string,
 			}
 			node.Status.NodeInfo.BootID = info.BootID
 
-			if utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
-				// TODO: all the node resources should use ContainerManager.GetCapacity instead of deriving the
-				// capacity for every node status request
-				initialCapacity := capacityFunc()
-				if initialCapacity != nil {
-					if v, exists := initialCapacity[v1.ResourceEphemeralStorage]; exists {
-						node.Status.Capacity[v1.ResourceEphemeralStorage] = v
-					}
+			// TODO: all the node resources should use ContainerManager.GetCapacity instead of deriving the
```

Comment:
should this TODO be resolved before GA?

Reply:
This TODO is not part of the feature. I think for ephemeral storage, it is already use getGapacity function.

this TODO was introduced here https://github.com/kubernetes/kubernetes/commit/596fa89af0353492c67015d6e053b2e8ac64a7a7

Need to talk to @mtaufen to understand more
mtaufen

Reply:
It's been a long time since I looked at this file, but I _think_ the TODO is saying that some of this logic is repeated and could be deduplicated, or maybe even that the result is already cached in ContainerManager and can just be retrieved. 

In any case it was a leftover TODO from a refactor, and shouldn't block anything.